### PR TITLE
Adding get_item_values call

### PR DESCRIPTION
**Pull request by dougblackjr**
Adds `get_item_values` call to the wrapper. Docs here: [https://developers.podio.com/doc/items/get-item-values-22365](https://developers.podio.com/doc/items/get-item-values-22365)

### CODE REVIEW

File: models/PodioItem.php [hunk 1]
```diff
@@ -301,4 +301,9 @@ public static function revert_to_revision($item_id, $revision, $attributes = arr
     return Podio::post("/item/{$item_id}/revision/{$revision}/revert_to", $attributes);
   }
 
+  public static function get_item_values($item_id)
```
Comment (daniel-sc) [inline models/PodioItem.php:304]:
```suggestion
  /**
   * @see https://developers.podio.com/doc/items/get-item-values-22365
   */
  public static function get_item_values($item_id)
```

**Review by daniel-sc — APPROVED** [review]
Thanks for the contribution! 
Except for the missing method comment (see suggestion) this looks fine for me.

**Comment by daniel-sc** [comment]
@dougblackjr Could you make a separate PR for the changes to PodioFile? There probably is some discussion necessary that could be separated from the initial change.

**Comment by dougblackjr** [comment]
@daniel-sc Done. Sorry about that.

**Review by rafaelmb — APPROVED** [review]
LGTM

**Comment by daniel-sc** [comment]
@dougblackjr thanks for the update! 
Would you need an immediate release or would you create the other PR before?

**Pull request merged**